x-pack/plugins/apm/server/lib/rum_client/get_js_errors.ts

```diff
@@ -76,12 +76,13 @@ export async function getJSErrors({
             },
             sample: {
               top_hits: {
-                _source: [
+                _source: ([
                   ERROR_EXC_MESSAGE,
                   ERROR_EXC_TYPE,
                   ERROR_GROUP_ID,
                   '@timestamp',
-                ],
+                  // FIXME: TopHitsAggregation._source is incorrect
+                ] as unknown) as string,
```

Comment:
not sure, how this is wrong, top_hits _source can be array,

Reply:
Yeah, the type from the client is wrong

Reply:
listed in https://github.com/elastic/elastic-client-generator/issues/197. I will update it to `@ts-expect-error @elastic/elasticsearch`

Reply:
already unnecessary. removed